### PR TITLE
[stable/airflow] Added option to mount dag volume as read-only in web deployment

### PR DESCRIPTION
**Pull request by michieldhadamus**
**What this PR does / why we need it:**
By default, webserver can write to dag volumes (and edit dags). We want to avoid this behaviour by mounting the dags volume as read-only. Compilation of dag.py to dag.pyc will be done by scheduler.

**Checklist**
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

 [x] DCO signed
 [x] Chart Version bumped
 [x] Variables are documented in the README.md
 [x] Title of the PR starts with chart name (e.g. [stable/chart])

### CODE REVIEW

**Comment by zanhsieh** [comment]
@michieldhadamus 
Would you mind to resolve the conflict file please? stable/airflow/Chart.yaml

**Comment by michieldhadamus** [comment]
@zanhsieh, resolved merge conflict

File: stable/airflow/templates/deployments-web.yaml [hunk 1]
```diff
@@ -119,6 +119,7 @@ spec:
               subPath: {{ .Values.persistence.subPath | default "" }}
           {{- else if .Values.dags.initContainer.enabled }}
             - name: dags-data
+              readOnly: { .Values.web.dagsReadOnly }
```
Comment (davidkarlsen) [inline stable/airflow/templates/deployments-web.yaml]: typo

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by michieldhadamus** [comment]
Messed up commit signing, will make a new PR later

**Pull request closed without merging**